tools/cloud_functions/gcs_event_based_ingest/gcs_ocn_bq_ingest/monitoring/monitoring_tables_and_views.sql

```diff
@@ -0,0 +1,105 @@
+/*
```

Comment:
please move this monitoring directory to gcs_event_based_ingest.
otherwise we will package these files into the cloud function on each deploy (for no reason as the cloud function does not use these files).
Rather than adding `.gcloudignore` it would be cleaner to have this live in the gcs_event_based_ingest/monitoring level because this has no relation to the python source code in the gcs_ocn_bq_ingest module